src/seedu/addressbook/AddressBook.java

```diff
@@ -162,7 +162,7 @@
     /*
      * NOTE : =============================================================================================
      * Note that the type of the variable below can also be declared as List<String[]>, as follows:
-     *    private static final List<String[]> ALL_PERSONS = new ArrayList<>()
+     *    private static final List<HashMap<String, String> ALL_PERSONS = new ArrayList<>()
```

Comment:
Good. 

When you modify the behavior, remember to update documentation and tests as well.